Update docs for 0.15 DB Layers

gewenyu99 opened the pull request:
This pull requests updates documentation in preparation for 0.15 DB Layers changes.

lohanidamodar commented on app/views/docs/database.phtml:
```diff
@@ -1,22 +1,35 @@
 <p>
     The Database Service allows you to store your application and users' data and fetch it using different supported queries.
```
The `Database` service should now be `Databases` everywhere

gewenyu99 replied:
fixed

lohanidamodar commented on app/views/docs/database.phtml:
```diff
@@ -1,22 +1,35 @@
     as well as a flexible permissions mechanism to allow you to easily segment data between different users, teams, and roles.
 </p>
 
+<h2><a href="/docs/database#database" id="database">Create Your Database</a></h2>
```
`/docs/databases/#database`

* Reminder, Appwrite homepage repo might need updates too with renaming links
* Other places might have this link, where we need update too

gewenyu99 replied:
fixed

lohanidamodar reviewed (changes requested):
Looking great, few changes.

eldadfux commented:
We should also update realtime docs for new database event names:
https://appwrite.io/docs/realtime

eldadfux commented:
Same here: https://appwrite.io/docs/webhooks#events

gewenyu99 commented:
@lohanidamodar I need a hand on a few things:
- What are the new events we introduced?
- What are the new channels we introduced?
- Is 0.15.x stable to build from source? I want to update the example response in realtime guide
- We should also document new functionality introduced in CLI, is there a version of the CLI that we generated which I can reference?

gewenyu99 commented:
This is related to:
https://github.com/appwrite/homepage/pull/44

lohanidamodar commented:
> @lohanidamodar I need a hand on a few things:
> 
> * What are the new events we introduced?
> * What are the new channels we introduced?
> * Is 0.15.x stable to build from source? I want to update the example response in realtime guide
> * We should also document new functionality introduced in CLI, is there a version of the CLI that we generated which I can reference?

1. All existing database related events will be prefixed by `databases.databaseId.`, new database related events `databases.*`, `databases.databaseId`, `databases.databaseId.create`, `databases.databaseId.read`, `databases.databaseId.update` and `databases.databaseId.delete`.
2. All database related channels will be prefixed by `databases.databaseId.`
3. 0.15.x should be stable to build from source as far as I remember
4. @christyjacob4  or @Meldiron  should know more about CLI

lohanidamodar commented on app/views/docs/storage.phtml:
```diff
@@ -3,7 +3,7 @@
 <h2><a href="/docs/storage#bucket" id="bucket">Buckets</a></h2>
 
-<p>With version 0.13 of Appwrite, we also have introduced storage buckets. Storage buckets are similar to collections we have in our <a href="/docs/database">Database</a> service. The difference is buckets also provide more power to decide what kinds of files, what sizes you want to allow in that bucket, whether or not to encrypt the files, scan with antivirus, and more. Let's look at how we can  ...
+<p>With version 0.13 of Appwrite, we also have introduced storage buckets. Storage buckets are similar to collections we have in our <a href="/docs/databases">Database</a> service. The difference is buckets also provide more power to decide what kinds of files, what sizes you want to allow in that bucket, whether or not to encrypt the files, scan with antivirus, and more. Let's look at how we can ...
```
```suggestion
<p>With version 0.13 of Appwrite, we also have introduced storage buckets. Storage buckets are similar to collections we have in our <a href="/docs/databases">Databases</a> service. The difference is buckets also provide more power to decide what kinds of files, what sizes you want to allow in that bucket, whether or not to encrypt the files, scan with antivirus, and more. Let's look at how we can create a bucket and configure it to your needs.</p>
```

lohanidamodar reviewed (approved):
LGTM
